vllm/model_executor/layers/attention/attention.py

```diff
@@ -17,11 +17,27 @@ class Attention(nn.Module):
 
     This class takes query, key, and value tensors as input. The input tensors
     can either contain prompt tokens or generation tokens.
+
+
+    If the input tensors contain prompt tokens, the layout is as follows:	
```

Comment:
Yeah agreed with the direction. But why don't we do this later when we actually have the abstraction? Right now, it will just require duplicated comments which is prone to error.